documentation/asciidoc/topics/con_hotrod_clients.adoc

```diff
@@ -12,3 +12,8 @@ Hot Rod clients running on {k8s} can access internal IP addresses for {brandname
 The default intelligence, `HASH_DISTRIBUTION_AWARE`, is recommended because it allows clients to route requests to primary owners, which improves performance.
 
 Hot Rod clients running outside {k8s} must use `BASIC` intelligence.
+
+.Encryption
+
+By default, {ispn_operator} uses TLS encryption to secure connections to the Hot Rod endpoint.
```

Comment:
thanks for the clarification. I've since removed this section about encryption. I think the main thing to note is that clients outside OpenShift/k8s need to get the TLS cert from the project and create a trust store. I've added that as a comment to those specific configuration examples to make it clear there.